<<<<<<< HEAD:tensorflow-base-cpu/0012-set-default-kmp-variables.patch
From 5fb5cc2e59c53ab61654ab8adb6c06b84c93ab9f Mon Sep 17 00:00:00 2001
=======
From 6a23bc9ad50a7c6f983a29c4fa8cb3fed04ec5ce Mon Sep 17 00:00:00 2001
>>>>>>> tf-base-cpu: update patches:tensorflow-base-cpu/0007-set-default-kmp-variables.patch
From: Jonathan Helmus <jjhelmus@gmail.com>
Date: Wed, 3 Apr 2019 11:34:43 -0500
Subject: [PATCH 7/7] set default kmp variables

---
 tensorflow/api_template_v1.__init__.py | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/tensorflow/api_template_v1.__init__.py b/tensorflow/api_template_v1.__init__.py
<<<<<<< HEAD:tensorflow-base-cpu/0012-set-default-kmp-variables.patch
index a83ff3a1..fa4c8ba5 100644
=======
index a83ff3a16c..fa4c8ba569 100644
>>>>>>> tf-base-cpu: update patches:tensorflow-base-cpu/0007-set-default-kmp-variables.patch
--- a/tensorflow/api_template_v1.__init__.py
+++ b/tensorflow/api_template_v1.__init__.py
@@ -18,6 +18,12 @@ from __future__ import absolute_import as _absolute_import
 from __future__ import division as _division
 from __future__ import print_function as _print_function
 
+# Set reasonable KMP defaults for MKL-DNN
+import os
+os.environ["KMP_BLOCKTIME"] = os.environ.get("KMP_BLOCKTIME", "0")
+os.environ["KMP_AFFINITY"] = os.environ.get("KMP_AFFINITY", "granularity=fine,verbose,compact,1,0")
+del os
+
 import distutils as _distutils
 import inspect as _inspect
 import os as _os
-- 
<<<<<<< HEAD:tensorflow-base-cpu/0012-set-default-kmp-variables.patch
2.21.0 (Apple Git-120)
=======
2.17.1
>>>>>>> tf-base-cpu: update patches:tensorflow-base-cpu/0007-set-default-kmp-variables.patch

